### PR TITLE
Xpanse create list fix

### PR DESCRIPTION
**Pull request by tomer-pan**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/XSUP-18770

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/ExpanseV2/Playbooks/playbook-Expanse_Load_Create_List.yml [hunk 1]
```diff
@@ -245,12 +285,14 @@ tasks:
...
+  "8":
+    id: "8"
+    taskid: 9228e6e2-2e0f-493c-8963-397b35f924bc
     type: regular
     task:
-      id: 62dda88b-b3a5-44c2-8fba-30516903d664
+      id: 9228e6e2-2e0f-493c-8963-397b35f924bc
       version: -1
       name: Set Empty List
       description: commands.local.cmd.list.set
```
Comment (melamedbn) [inline Packs/ExpanseV2/Playbooks/playbook-Expanse_Load_Create_List.yml:298]: This is the OOTB description of this automation?

File: Packs/ExpanseV2/Playbooks/playbook-Expanse_Load_Create_List.yml [hunk 2]
```diff
@@ -1,5 +1,7 @@
 id: Expanse Load-Create List
 version: -1
+contentitemexportablefields:
+  contentitemfields: {}
 name: Expanse Load-Create List
 description: |
   Sub-playbook to support Expanse Handle Incident playbook.
```
Comment (melamedbn) [inline Packs/ExpanseV2/Playbooks/playbook-Expanse_Load_Create_List.yml:7]: Maybe change the description to be more specific?

**Review by melamedbn — APPROVED** [review]

**Comment by ShirleyDenkberg** [comment]
@melamedbn Reviewed RN.

File: Packs/ExpanseV2/Playbooks/playbook-Expanse_Load_Create_List.yml (hunk 1 above)
Comment (tomer-pan) [reply Packs/ExpanseV2/Playbooks/playbook-Expanse_Load_Create_List.yml:298]: I think that it's a server script and this is why the description looks like that

File: Packs/ExpanseV2/Playbooks/playbook-Expanse_Load_Create_List.yml (hunk 2 above)
Comment (tomer-pan) [reply Packs/ExpanseV2/Playbooks/playbook-Expanse_Load_Create_List.yml:7]: It is a multiple line description, maybe you missed the lower lines?

**Pull request merged**